rio_cogeo/__init__.py

```diff
@@ -1,3 +1,6 @@
 """rio_cogeo: Cloud Optimized GeoTIFF creation and validation plugin for rasterio."""
 
+from rio_cogeo.cogeo import cog_info, cog_translate, cog_validate  # noqa
```

Comment:
@kylebarron should we remove the `cog` prefix for top-levels ?

```python
from rio_cogeo.cogeo import info as info
from rio_cogeo.cogeo import cog_translate as translate
from rio_cogeo.cogeo import cog_validate as validate
```

Reply:
I'd suggest not. I think it would be more confusing for users to have the same function be renamed.